README.md

```diff
@@ -71,13 +71,15 @@ To use plugin you should add these lines to configuration file `.jscsrc`:
 
 ### checkAnnotations
 
-Ensures tag names are valid
+Checks tag names are valid
 
 There are 3 presets for `Closure Compiler`, `JSDoc3` and `JSDuck5`.
 
-By default it allows any tag of mixed set. You can pass `Object` to select preset with `preset` field and add custom tags with `extra` field.
+By default it allows any tag of mixed set. You can pass `Object`
```

Comment:
What did we say about this again? `By default it allows any tag of mixed set` I don't think mixed is that clear - maybe something like `any tag from the set of presets` or `any tag from the 3 presets`?